### PR TITLE
Add translations for Announcements

### PR DESCRIPTION
**Pull request by pixeldesu**
Considering this was new-ish and I specifically gave the order not to localize the strings when implementing, this is now done.

Adding locales for `Announcement`/`AnnouncementController` using the Rails scoping features, akin to

```yml
en:
  controller:
    action:
      message: "Test"
```

So that following works

```rb
def action
  t(".message")
end
```

----

**Testing:**
* Check the views in `/admin/announcements` for proper text/translation display

### CODE REVIEW

File: config/locales/activerecord.en.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+      announcement:
+        one: "Announcement"
+        other: "Announcements"
+    attributes:
+      announcement:
+        content: "Content"
+        link_href: "Link URL"
+        link_text: "Link text"
+        starts_at: "Start time"
+        ends_at: "End time"
```
Comment (nilsding) [inline config/locales/activerecord.en.yml]: Final newline missing.

**Review by nilsding — CHANGES_REQUESTED** [review]

File: config/locales/en.yml [hunk 2]
```diff
@@ -457,6 +457,27 @@ en:
           invalid_code: "The code you entered was invalid."
         setup:
           success: "Two factor authentication has been enabled for your account."
+  announcement:
+    index:
+      title: :activerecord.models.announcement.other
+      confirm: "Are you sure you want to delete this announcement?"
+      new: "Add Announcement"
```
Comment (raccube) [inline config/locales/en.yml:464]:
```suggestion
      new: "Make an Announcement"
```

**Review by raccube — CHANGES_REQUESTED** [review]

File: config/locales/en.yml (hunk 2 above)
Comment (raccube) [reply config/locales/en.yml:464]: 🥚

Comment (pixeldesu) [reply config/locales/en.yml:464]: 🛑

**Review by raccube — APPROVED** [review]

**Review by nilsding — APPROVED** [review]

**Comment by nilsding** [comment]
# **🐾🐾🐾🐾🐾**

**Pull request merged**